### PR TITLE
Blog rewrite to React / GraphQL

### PR DESCRIPTION
**Pull request by broskoski**
rewritten-blog

The only not-nice thing here is that there is no automatic type generation, due to the different endpoint.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -141,6 +143,7 @@
     "jeet": "*",
     "jquery": "*",
     "jquery.finger": "*",
+    "lodash": "^4.17.20",
```
Comment (dzucconi) [inline package.json:146]: I'd remove this and just use the native method

Comment (dzucconi) [reply package.json:146]: (Remember we don't have bundle splitting yet)

Comment (damassi) [reply package.json:146]: Second @dzucconi on that, but also note that https://github.com/lodash/babel-plugin-lodash will pluck things out if you just _have to_ lol

File: src/v2/apollo/index.tsx [hunk 2]
```diff
@@ -31,6 +32,9 @@ const {
 
 const clientHttpLink = new BatchHttpLink({ uri: CLIENT_GRAPHQL_ENDPOINT })
 const serverHttpLink = new BatchHttpLink({ uri: GRAPHQL_ENDPOINT })
+const contentfulHttpLink = createHttpLink({
+  uri: 'http://localhost:5000/graphql/contentful',
```
Comment (dzucconi) [inline src/v2/apollo/index.tsx:36]: Needs the ENV var?

Comment (broskoski) [reply src/v2/apollo/index.tsx:36]: Yes, thank ye

**Comment by damassi** [comment]
Funny how easy it can be... Curious how it works on the CMS side? Contentful calls a mutation to the GraphQL server which then reads in the data on the site?

**Comment by broskoski** [comment]
@damassi contentful just offers a graphql api now (in addition to rest). they have their own CMS. Am I understanding your q?

**Comment by damassi** [comment]
Ahh i see -- was thinking that it was a headless CMS that still required the user to host content themselves. Nice 👍

**Comment by broskoski** [comment]
@damassi oh yea, I'm just making our own graphql endpoint to proxy through.

**Pull request merged**